doc/overview/grass_overview.rst

```diff
@@ -74,6 +74,4 @@ Details
 @QUICKSTART_grass@
 
 .. presentation-note
-    GRASS GIS provides powerful raster, vector, and geospatial processing. It includes tools for spatial modeling, visualization of raster and vector data, management and analysis of geospatial data, and the processing of satellite and aerial imagery. It also provides the capability to produce sophisticated presentation graphics and hardcopy maps.
-    It includes over 400 built-in analysis modules and 100 community supplied modules and toolboxes.
-    With over 30 years of continuous development, GRASS is both the oldest and largest Open Source GIS available. It is capable of very powerful analysis, but may not be as simple to get started with as other offerings with more of a geodata viewer focus. Many Open Source projects make use of GRASS's algorithms.
+    GRASS provides powerful raster and vector processing. It includes hundreds of tools for spatial modeling, visualization and analysis.
```

Comment:
I think we should keep more text from the original.

Reply:
I've added more words. Still less than before. I'm very aware of how hard it is to trim down to something that can be presented in a lightning talk.